llmproxy/models/llama2.py

```diff
@@ -0,0 +1,63 @@
+import requests
+from llmproxy.models.base import BaseChatbot, CompletionResponse
+from llmproxy.utils.enums import BaseEnum
+
+class Llama2Model(str, BaseEnum):
+    LLAMA_2_7B="Llama-2-7b-chat-hf"
+    LLAMA_2_13B="Llama-2-13b-chat-hf"
+    LLAMA_2_70B="Llama-2-70b-chat-hf"
+
+
+
+class Llama2(BaseChatbot):
+
+    def __init__(
+        self,
+        prompt: str = "",
+        system_prompt: str = "Answer politely",
+        api_key: str = "",
+        temperature: float = 1.0,
+        model: Llama2Model = Llama2Model.LLAMA_2_7B
```

Comment:
This is a bug, please refer to victor's mistral branch, or the the openai file in the main branch. 

(It should be Llama2Model.LLAMA_2_7B.value